### ISSUE TITLE
Rule proposal: one prop per line

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#13]
It might be nice to enforce only having one prop per line.

Good:

```jsx
<MyComponent
  firstProp="value"
  secondProp="value"
/>
```

Bad:

```jsx
<MyComponent firstProp="value" secondProp="value" />
```

Or maybe it could be configured to only apply to multiline components. In which case:

Good:

```jsx
<MyComponent firstProp="value" secondProp="value" thirdProp="value" />
```

Bad: 

```jsx
<MyComponent firstProp="value" secondProp="value"
  thirdProp="value" />
```
